src/registry/registry.py

```diff
@@ -0,0 +1,71 @@
+class Registry:
+    # !TODO may be we need to change the name of class to TorchokModule
+    """ Class for generate Deep Learning modules.
+    
+    Each module can write to the register memory (dict) classes types that need to be remembered.
```

Comment:
The registry is meant to be used as a decorator for any classes, so that they can be accessed by class name for instantiating. Example:
```
COMPONENTS = Registry('components')
@COMPONENTS.register_class
class SomeComponent:
    ...
```